test/hilbert/test_hilbert.py

```diff
@@ -675,7 +683,7 @@ def test_tensor_combination():
     assert len(hit._hilbert_spaces) == 5
     assert isinstance(repr(hit), str)
 
-    hi3 = nk.experimental.hilbert.Particle(N=5, geometry=geo_default)
```

Comment:
I would feel more confident if those tests which are now of experimental features where moved to a different file.

As I don't trust 100% AI to not do random changes here and there, at least it's easy to check that they do not change the tests of other features to make sure they pass.